examples/hyperparameter-optimisation.ipynb

```diff
@@ -121,7 +121,7 @@
    "outputs": [],
    "source": [
     "from skopt import gp_minimize\n",
-    "res_gp = gp_minimize(objective, space, maxiter=50, random_state=0)"
+    "res_gp = gp_minimize(objective, space, n_random_start=1, n_model_iter=50, random_state=0)"
```

Comment:
Could you still update this notebook? Note sure when #119 will be ready, but your PR clearly is. In the meantime, we should not break things.